data/os/windows.yaml

```diff
@@ -6,8 +6,8 @@ mcollective::libdir: "C:/ProgramData/PuppetLabs/mcollective/plugins"
 mcollective::configdir: "C:/ProgramData/PuppetLabs/mcollective/etc"
 mcollective::rubypath: 'C:\Program Files\Puppet Labs\Puppet\sys\ruby\bin\ruby.exe'
 
-mcollective::config:
-  libdir: "%{hiera('mcollective::libdir')}/mcollective"
+mcollective::common_config:
+  libdir: "%{hiera('mcollective::libdir')}"
```

Comment:
for the moment we have to hardcode/duplicate this due to https://tickets.puppetlabs.com/browse/PUP-7061

so `libdir` here should be duplicate of mcollective::libdir :(